pom.xml

```diff
@@ -176,7 +181,7 @@
         <dependency>
             <groupId>org.igniterealtime.openfire</groupId>
             <artifactId>xmppserver</artifactId>
-            <version>4.2.3</version>
+            <version>4.7.4</version>
```

Comment:
Does this affect backwards compatibility much? I'm not to bothered by not supporting Openfire 4.2, but have you tested this with, say, Openfire 4.7.3? If that works - fine. If it doesn't, then maybe we should consider depending on something like 4.7.0 instead of 4.7.4.